src/main/java/ui/components/NavigableListView.java

```diff
@@ -45,7 +46,9 @@
     // (when nothing is selected, both should be no-ops)
     private boolean saveSelectionCalled = false;
 
-    private IntConsumer onItemSelected = i -> {};
+    public static final boolean IS_PRIMARY_KEY = true;
+
+    private BiConsumer<Integer, Boolean> onItemSelected = (index, wasRightClick) -> {};
```

Comment:
The name `wasRightClick` is kind of inconsistent with how it works (the value is true only if it was *not* a right click).